app/src/ui/changes/changes-list.tsx

```diff
@@ -49,7 +49,8 @@ interface IChangesListProps {
   readonly onDiscardChanges: (file: WorkingDirectoryFileChange) => void
   readonly askForConfirmationOnDiscardChanges: boolean
   readonly onDiscardAllChanges: (
-    files: ReadonlyArray<WorkingDirectoryFileChange>
+    files: ReadonlyArray<WorkingDirectoryFileChange>,
+    dontDisplayAsAll?: boolean
```

Comment:
blocking: i'm having a hard time reasoning about this variable as its named. could we rename this to be a positively phrased name? like `displayAsAll` or `isDiscardAll` or something like that?

Reply:
Great point! How about `isDiscardingAllChanges`?
It's a little bulky, but I think the clarity would be worth it.